docs/querying/virtual-columns.md

```diff
@@ -80,4 +82,106 @@ The expression virtual column has the following syntax:
 |--------|-----------|---------|
 |name|The name of the virtual column.|yes|
```

Comment:
This table should include the `type` property as required. Description can be something like "Type of the virtual column. Set to `expression` for an expression virtual column."

Reply:
Ditto for the `nested-field` virtual column and `mv-filtered` virtual column